x-pack/plugin/mapper-constant-keyword/src/main/java/org/elasticsearch/xpack/constantkeyword/mapper/ConstantKeywordFieldMapper.java

```diff
@@ -264,21 +265,16 @@ protected void parseCreateField(ParseContext context) throws IOException {
     }
 
     @Override
-    public List<String> lookupValues(SourceLookup lookup, String format) {
+    public ValueFetcher valueFetcher(MapperService mapperService, String format) {
         if (format != null) {
             throw new IllegalArgumentException("Field [" + name() + "] of type [" + typeName() + "] doesn't support formats.");
         }
 
-        return fieldType().value == null
+        return lookup -> fieldType().value == null
```

Comment:
`fieldType.value == null ? lookup -> List.of() : lookup -> List.of(fieldType.value)`? It isn't really important, but it'd make me feel good.

Reply:
👍  will update this, feeling good is worth it !